### ISSUE TITLE
ERROR! no action detected in task

### ISSUE THREAD

**Issue by gordonmurray** [ansible/awx#158]
##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME
 - Installer

##### SUMMARY
From the installer folder, performing: ansible-playbook -i inventory install.yml 

Results in: 

ERROR! no action detected in task
The offending line appears to be:

- name: Build AWX distribution
  ^ here

##### ENVIRONMENT

* AWX version: 2dda863baf72282ddff6c45cb1f4d7f3175ea733
* Ansible version:  2.0.0.2
* Operating System: Ubuntu 16.04

##### STEPS TO REPRODUCE

Clone the awx repository
Navigate to the installer directory
Run: ansible-playbook -i inventory install.yml

##### EXPECTED RESULTS

AWX to begin installation.

##### ACTUAL RESULTS

Ansible error:

ERROR! no action detected in task
The offending line appears to be:

- name: Build AWX distribution
  ^ here

**Comment by ajvn** [comment]
Affected by this as well.

**Comment by matburt** [comment]
This Ansible version is too old, please try with a newer version of Ansible.

**Comment by ajvn** [comment]
@matburt Thank you Mathew.

**Comment by gordonmurray** [comment]
Thank you @matburt Confirmed, I updated to Ansible 2.3.2.0 and it has started installing smoothly.

**Comment by steaksauce-** [comment]
Just found this and updated. As expected, it worked. The Ansible version included in the default Ubuntu repos for Ubuntu 16.04 is 2.0.0.2 as of 01/11/2018.

It's recommended to use the Ansible ppa to install:
```
# remove older ansible version
sudo apt-get remove ansible

# add ppa and install new ansible version 
sudo apt-add-repository ppa:ansible/ansible
sudo apt-get update
sudo apt-get install ansible
```

My playbook ran great after doing this!

**Comment by ketansp** [comment]
@steaksauce- this indeed solved the problem. I had an ansible script which was working fine on another centos machine. When I migrated this deploy process to newer ubuntu machine, it started giving me error `ERROR! no action detected in task`. Upgrading ansible installable from ansible repo solved this problem for me.

Thank you.
